README.md

```diff
@@ -1,57 +1,53 @@
 # gulp-kss
 
-Gulp plugin for KSS ([Knyle Stype Sheets](http://warpspire.com/kss/)) documentation generation.
+Gulp plugin for KSS ([Knyle Style Sheets](http://warpspire.com/kss/)) documentation generation.
 
-This plugin is based on [kss-node](https://github.com/hughsk/kss-node) and generates a styleguide based on code documentation. The plugin is mainly a fork of `kss-nodes`'s bin script, not how a beautiful gulp plugin should look like, but it works. 
-
-This plugin currently lacks tests.
+This plugin is based on [kss-node](https://github.com/hughsk/kss-node) and generates a styleguide based on code documentation. It builds upon the original [gulp-kss](https://github.com/philj/gulp-kss) by PhilJ but integrates newer node-kss features including custom properties, handlebars helpers, and more.
 
 ## Install
 
 ```
-npm install gulp-kss
+git clone https://github.com/kedruff/gulp-kss.git
+npm install
```

Comment:
Good catch. My readme.md is a little bit of an awkward amalgamation of my *public* github version and the one that resides on the enterprise Github repo that we have internally at my employer.